web/src/stream_create.js

```diff
@@ -428,7 +428,7 @@ export function set_up_handlers() {
 
         if (principals.length >= 50) {
             const html_body = render_subscription_invites_warning_modal({
-                stream_name,
+                name: stream_name,
```

Comment:
I've updated these template placeholder variables in a few cases. If the string clearly has "channel" in it, then I've usually changed "stream_name" to just be "name". If it's less clear, then I've changed it to be "channel_name".

Reply:
I think sticking with `channel_name` might be better if it's only a few cases, just in that the JS syntax of just passing a parameter by its name is very handy.

Reply:
Done. There are only 5 or 6 of these in the `web/` code.